tests/analysis/test_ols_analysis.py

```diff
@@ -16,3 +17,10 @@ def test_get_pvalue():
     analysis_df_full = pd.concat([analysis_df for _ in range(100)])
     analyser = OLSAnalysis()
     assert analyser.get_pvalue(analysis_df_full) >= 0
+
+
+@pytest.mark.parametrize("hypothesis", ["one_sided", "two_sided"])
+def test_get_pvalue_hypothesis(hypothesis):
```

Comment:
I would add tests for the pvalue_based_on_hypothesis method too, since it has some logic